twisted.web.resource.Resource: use "new" (old)-style classes for python 2
--- a/src/twisted/web/resource.py
+++ b/src/twisted/web/resource.py
@@ -101,7 +101,7 @@
 
 
 @implementer(IResource)
-class Resource:
+class Resource(object):
     """
     Define a web-accessible resource.
 
